lib/ViewMetaData/ViewMetaData.js

```diff
@@ -1,37 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import { isEqual } from 'lodash';
+
 import { MetaSection } from '@folio/stripes-components';
+import { stripesConnect } from '@folio/stripes-core';
 
 class ViewMetaData extends React.Component {
   static manifest = Object.freeze({
-    createdBy: {
+    updaters: {
       type: 'okapi',
       records: 'users',
-      path: 'users?query=(id==!{metadata.createdByUserId})',
-      fetch: false,
-      accumulate: true,
-    },
-    updatedBy: {
-      type: 'okapi',
-      records: 'users',
-      path: 'users?query=(id==!{metadata.updatedByUserId})',
-      fetch: false,
-      accumulate: true,
+      path: 'users',
+      GET: {
+        params: {
+          query: (queryParams, pathComponents, resourceValues) => {
+            if (resourceValues.updaterIds && resourceValues.updaterIds.length) {
+              return `(${resourceValues.updaterIds.join(' or ')})`;
+            }
+            return null;
+          },
+        }
+      }
     },
+    updaterIds: [],
```

Comment:
Is this supposed to be an `{}` or is this some syntax I don't know about?

Reply:
It's supposed to be an empty array, but in fact local resources are either objects or strings, so `{}` is correct.